### DISCUSSION TITLE
Rewind/fastforward with physical buttons (example back/prev on bluetooth headphones)

### DISCUSSION THREAD

**Discussion by murimaa** [LeoKlaus/plappa#60] [Feature Requests]
Hi,
I really like the app! It inspired me to start self hosting and managing my books on my home server with audiobookshelf.

One thing holding be back though, is that back/prev buttons on bluetooth headphones (or physical buttons in car steering wheel for example) don't do anything. I would like to be able to rewind 15s with the prev button. I use it a lot with other audio book apps to relisten to a sentence, or just rewind a bit. 

(Even better if it was also possible to customize and remap volume up/down to skip backwards/forwards instead of controlling volume, as some headphones only have volume controls. Don't know if its possible though :)

Thanks for the great work!

**Comment by LeoKlaus** [comment]
Hey murimaa, sorry for the late response.

I've fixed this with one of the recent updates (I think it was 1.1.3), the duration of the skip is also configurable in the settings.

> Even better if it was also possible to customize and remap volume up/down to skip backwards/forwards instead of controlling volume, as some headphones only have volume controls. 

That's not possible as far as I know. The skip actions can have user-defined actions but AFAIK, the volume control is handled by the OS and not passed through to an app.
